{"uid":"c146d3205ff1b009","name":"test_edit_account_empty_data","fullName":"test.Cuentas.test_edit_account#test_edit_account_empty_data","historyId":"65e07365c4ff5794c49b7f8fa4aff299","time":{"start":1726777529345,"stop":1726777529682,"duration":337},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError","statusTrace":"setup_edit_account = ({'Espo-Authorization': 'YWRtaW46ODAwNDkwNTM=', 'X-Api-Key': '4c29fe9482f3bccb7ac0a7d04015a2e7'}, {'assignedUserId': None, 'assignedUserName': None, 'billingAddressCity': None, 'billingAddressCountry': None, ...})\n\n    @pytest.mark.regression\n    @pytest.mark.functional\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_edit_account_empty_data(setup_edit_account):\n        headers, account = setup_edit_account\n        payload_account = PayloadAccount().build_payload_add_account(name=\"prueba nueva\", billingAddressCity=\"prueba\",\n                                                                     billingAddressCountry=\" \",\n                                                                     billingAddressPostalCode=\" \",\n                                                                     billingAddressState=\" \",\n                                                                     billingAddressStreet=\" \",\n                                                                     description=\"prueba\", emailAddress=\"prueba@prueba.com\",\n                                                                     industry=\"Architecture\", phoneNumber=\"+15555555555\",\n                                                                     shippingAddressCity=\"prueba\",\n                                                                     shippingAddressCountry=\"prueba\",\n                                                                     shippingAddressPostalCode=\"prueba\",\n                                                                     shippingAddressState=\"prueba\",\n                                                                     shippingAddressStreet=\"prueba\", type=\"Customer\",\n                                                                     website=\"prueba.com\")\n        response = EspocrmRequest().put(EndpointAccount.view(account['id']), headers, payload_account)\n>       AssertionStatusCode().assert_status_code_400(response)\n\ntest/Cuentas/test_edit_account.py:97: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n>       assert response.status_code == 400\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:19: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1726777492492,"stop":1726777492493,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_edit_account","time":{"start":1726777528990,"stop":1726777529345,"duration":355},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"setup_edit_account::0","time":{"start":1726777529698,"stop":1726777530017,"duration":319},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"functional"},{"name":"tag","value":"regression"},{"name":"parentSuite","value":"test.Cuentas"},{"name":"suite","value":"test_edit_account"},{"name":"host","value":"fv-az698-838"},{"name":"thread","value":"2084-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.Cuentas.test_edit_account"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["functional","regression","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"c146d3205ff1b009.json","parameterValues":[]}